### PR TITLE
Add missing links to team pages #39

### PR DESCRIPTION
**Pull request by LeoAlvarenga**
I made some change as requested on issue #39, i tried to follow the images on issue post, if i did something wrong, let me know!

### CODE REVIEW

**Comment by LeoAlvarenga** [comment]
if possible, lable it as valid to hacktoberfest

**Review by humancalico — CHANGES_REQUESTED** [review]
Looks great @LeoAlvarenga. Just a minor nitpick, can you use an icon or a symbol like `←`/`→`  instead of `<--`/`-->`?

**Comment by DelusionalOptimist** [comment]
@LeoAlvarenga Thanks for working on this issue. Glad you understood what I was trying to say. :grin:
Adding to the changes requested by @humancalico, can you structure the pages in a format like this:-

Links on [team](https://jodc.vercel.app/team) page:- 
&larr;`2018-2019` &larr;`2019-2020`

Links on [team-2019](https://jodc.vercel.app/team-2019) page:-
&larr;`2018-2019` `2020-2021`&rarr;

Links on [team-2018](https://jodc.vercel.app/team-2018) page:-
`2019-2020`&rarr; `2020-2021`&rarr;

Basically this is like browsing pages of any site.

**Comment by LeoAlvarenga** [comment]
Sure! i'll try to do those changes as soon as possible

**Comment by LeoAlvarenga** [comment]
I just did the changes requested by you guys, hope you like!

**Comment by humancalico** [comment]
Hey @LeoAlvarenga, can you fix these merge conflicts?

**Comment by humancalico** [comment]
cc @DelusionalOptimist

**Comment by LeoAlvarenga** [comment]
Sure, I'll try to solve it

**Comment by LeoAlvarenga** [comment]
I think it's all set now!

File: src/routes/team-2018.svelte [hunk 1]
```diff
@@ -21,26 +40,38 @@
...
+import { onMount } from 'svelte';
 import TeamComponent from '../components/TeamComponent.svelte';
-
-export let datas;
+let datas = [];
+onMount(async () => {
+  const res = await fetch('./Data/2018Team.json');
+  const textData = await res.text();
+  datas = await JSON.parse(textData);
+});
```
Comment (aayushmau5) [inline src/routes/team-2018.svelte:68]: We removed this.Please fetch the latest commits.

File: src/routes/team.svelte [hunk 2]
```diff
@@ -57,18 +57,17 @@
...
 </style>
 
-<script context="module">
-export async function preload() {
-  const res = await this.fetch('Data/currentTeam.json');
-  const datas = await res.json();
-  return { datas };
-}
-</script>
-
```
Comment (aayushmau5) [inline src/routes/team.svelte:67]: Again. Fetch the latest commits.

**Review by aayushmau5 — CHANGES_REQUESTED** [review]

File: src/routes/team-2019.svelte [hunk 3]
```diff
@@ -53,3 +83,11 @@ export let datas;
     />
   {/each}
 </section>
+
+<div class="other-team">
+  <p class="other-team-message">Previous Year Teams and Mentors</p>
+  <div class="year-container">
+    <a rel="prefetch" href="/team-2018"> &larr; 2018-2019 </a>
+    <a rel="prefetch" href="/team"> &rarr; 2020-2021 </a>
```
Comment (DelusionalOptimist) [inline src/routes/team-2019.svelte:91]: Minor change: this should be `<a rel="prefetch" href="/team"> 2020-2021 &rarr;</a>`

File: src/routes/team-2018.svelte (hunk 1 above)
Comment (LeoAlvarenga) [reply src/routes/team-2018.svelte:68]: I'm trying to, but git says i'm already up to date

**Review by aayushmau5 — APPROVED** [review]
LGTM.

**Review by DelusionalOptimist — APPROVED** [review]
LGTM. Thanks @LeoAlvarenga :+1:.
